vhdbuilder/packer/windows-image.env

```diff
@@ -19,16 +19,16 @@ WINDOWS_2022_GEN2_BASE_IMAGE_SKU=2022-datacenter-core-smalldisk-g2
 WINDOWS_2022_GEN2_BASE_IMAGE_VERSION=20348.2340.240303
 
 # CLI example to get the latest image version:
-#   az vm image show --urn MicrosoftWindowsServer:WindowsServer:23H2-Datacenter-Core-smalldisk:latest
-WINDOWS_23H2_BASE_IMAGE_SKU=23H2-Datacenter-Core-smalldisk
-WINDOWS_23H2_BASE_IMAGE_VERSION=25398.2113.231109
+#   az vm image show --urn "MicrosoftWindowsServer:WindowsServer:23h2-datacenter-core:latest
+WINDOWS_23H2_BASE_IMAGE_SKU=23h2-datacenter-core
```

Comment:
Is it expected that there is no smalldisk for 30gb images?

Reply:
Yes. It's expected.